### DISCUSSION TITLE
Does Harbor support 2 ingresses for same habor helm deployment?

### DISCUSSION THREAD

**Discussion by aathivignesh19** [goharbor/harbor#18739] [Q&A]
Hi team, Im working on a harbor setup with nginx ingress controller in which im configuring 2 ingresses for harbor. 1 with client cert auth adding annotation like - auth-tls-verify-client, auth-tls-secret and another without these annotations. Is this setup possible?

I tried this and when i try docker login via ingress without client cert auth i get "400 No required SSL certificate was sent" error

Or

Should i have harbor installed using helm in different namespace with another nginx ingress controller in that namespace with ingress endpoint and have both harbor deployments using same DB and S3 Bucket.

Any recommendation is much appreciated. Thanks.

**Comment by AllForNothing** [comment]
@zyyw Please help to have a look at this

**Reply by stonezdj** [reply]
No, it is not support multiple ingress, because the docker cli to pull image it requires access a single token service URL, the token service service is unique. if you configured urlA, then you should get token from `urlA/service/token`, if you configured B, you can't get token from `urlA/service/token`.

**Reply by aathivignesh19** [reply]
Thank you so much. I have got a follow up question. If i have a HarborA deployed with Ingress ControllerA - with cert auth ingress in namespace one and HaborB  deployed with Ingress ControllerB - without cert auth ingress in namespace two with both harbor helm deployment using same external DB, S3 bucket. Does it work? Or in addition to this i should have an external redis instance to support? Does that work with this setup?
